0 General/06 Programming Languages.md

```diff
@@ -2,14 +2,30 @@
 
 # Software
 
-- [Hardware and Machine Code](#hardware-and-machine-code)
-- [Low- and High-Level Languages](#low--and-high-level-languages)
-- [Types of Languages](#types-of-languages)
-  - [Programming Languages](#programming-languages)
-  - [Non-Programming Languages](#non-programming-languages)
-  - [Imperative vs Declarative](#imperative-vs-declarative)
-  - [Procedural vs Object-Oriented](#procedural-vs-object-oriented)
-- [Typing Systems](#typing-systems)
+- [Software](#software)
+	- [Hardware and Machine Code](#hardware-and-machine-code)
+	- [Low- and High-Level Languages](#low--and-high-level-languages)
```

Comment:
Should this be `Low-Level`? Also was this created using a tool? If so 'Low-' would appear elsewhere as well.

Reply:
Looks like it appears again on line 34. 

Leaving note here since it will not let me comment on that line.

Reply:
Matthew Cooper made most of the ToCs, I don't know if they were created using an automated tool or not. What is the built in ToC on GitHub? I'm 100% in favor of not needing to maintain a ton of ToCs. In general I think Matthew did overdo it with the ToCs a bit, they take up a lot of space and I don't exactly think our docs are that complicated to get through. Thoughts?

Reply:
At the top left of each doc there is a dropdown menu that will link to each header of a md file, it's pretty handy.

Reply:
Ooooooh I never noticed that! Yeah, unless there is some sort of automated tool for producing markdown ToCs I vote we don't bother to maintain those and just point students at the automated ToC instead